test/integration/visitor_can_add_items_to_cart_test.rb

```diff
@@ -104,4 +104,64 @@ class VisitorCanAddItemsToCart < ActionDispatch::IntegrationTest
     assert page.has_content?("Super yummy")
     assert page.has_content?("Total: $20")
   end
+
+  test "user can adjust the quantity of an item in the cart" do
+    skip
+    category_1 = Oil.create(name: "Lard")
+    category_2 = Oil.create(name: "Coconut Oil")
+
+    item_1 = Chip.create(name: "Slotachips", price: 20, description: "Super yummy", oil_id: category_1.id)
```

Comment:
Useless assignment to variable - `item_1`. Did you mean `item_2`?<br>Line is too long. [106/80]